LiteCore/Query/Translator/SelectNodes.cc

```diff
@@ -362,6 +378,7 @@ namespace litecore::qt {
                     }
                     addChild(_groupBy, group);
                 }
+                ctx.hasGroupBy = true;
```

Comment:
I don't think this flag is a good idea. The context already has a reference to the SelectNode, so just add a `bool hasGroupBy()` accessor to SelectNode.

Reply:
changed as suggested